Source/ACE.DatLoader/Entity/AnimationHooks/ReplaceObjectHook.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace ACE.DatLoader.Entity.AnimationHooks
+{
+    public class ReplaceObjectHook : IHook
+    {
+        public AnimationPartChange APChange { get; set; }
+
+        public static ReplaceObjectHook ReadHookType(DatReader datReader)
+        {
+            ReplaceObjectHook r = new ReplaceObjectHook();
+
+            AnimationPartChange ao = new AnimationPartChange();
+            ao.PartIndex = (byte)datReader.ReadUInt16();
```

Comment:
I didn't bother, since long term the goal is to remove this class and replace with the AnimationOverride object. I've got that on my to-do list (there's a few places this is being used currently that can be replaced so we just have the one object).

Perhaps an extension, like you suggested with the Positions, would a better solution.